tests/test_adjoint.py

```diff
@@ -33,36 +33,30 @@
 device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
 dtype = torch.get_default_dtype()
 
-ito_methods = {'euler': 'ito',
-               'milstein': 'ito',
-               'srk': 'ito'}
-stratonovich_methods = {'midpoint': 'stratonovich',
-                        'log_ode': 'stratonovich'}
```

Comment:
What's the rationale for removing Euler?

Reply:
It's just hard for gradients to match up.